README.md

```diff
@@ -31,13 +33,47 @@ osio {
 
     build resources: resources
     deploy resources: resources, env: 'stage'
-    
+
     // wait for user to approve the promotion to "run" environment
     deploy resources: resources, env: 'run', approval: 'manual'
   }
 }
 ```
 
+### deploy stand-alone application with external configuration
+
+The following example builds a nodejs booster and deploys it to a `stage` environment and then on approval to the `run` environment.
+
+It also loads an external resource like `configmap` and deploy it to `stage` and `run` environments
+
+```groovy
+@Library('github.com/fabric8io/osio-pipeline@master') _
+
+osio {
+    config runtime: 'node'
+
+    ci {
+        def app = processTemplate()
+        build app: app
+    }
+
+    cd {
+
+      def resources = processTemplate(params: [
+        release_version: "1.0.${env.BUILD_NUMBER}"
+      ])
+      def cm = loadResources(file: "configmap.yaml")
```

Comment:
Excellent!
Could you update the example to have a list?

Reply:
Yes @sbose78 :+1: